crypto/dilithium/pqcrystals_dilithium_ref_common/sign.h

```diff
@@ -4,17 +4,33 @@
 #include <stddef.h>
 #include <stdint.h>
 #include "params.h"
-#include "polyvec.h"
-#include "poly.h"
 
 int crypto_sign_keypair(ml_dsa_params *params, uint8_t *pk, uint8_t *sk);
 
+int crypto_sign_keypair_internal(ml_dsa_params *params,
+                                 uint8_t *pk,
+                                 uint8_t *sk,
+                                 const uint8_t *seed);
+
 int crypto_sign_signature(ml_dsa_params *params,
                           uint8_t *sig, size_t *siglen,
                           const uint8_t *m, size_t mlen,
                           const uint8_t *ctx, size_t ctxlen,
                           const uint8_t *sk);
 
+int crypto_sign_signature_deterministic(ml_dsa_params *params,
```

Comment:
I don't see the deterministic variants used anywhere, why are we adding them?

Reply:
Removed in [aada875](https://github.com/aws/aws-lc/pull/1999/commits/aada875da98c6f173d11a001195b81f47cdcc451)